libbeat/docs/dashboards.asciidoc

```diff
@@ -102,6 +102,10 @@ predefined +{beatname_lc}-*+ index pattern and set it as the default.
 
 image:./images/kibana-created-indexes.png[Kibana configured indexes]
 
+NOTE: After importing the index-pattern in Kibana, you need to refresh each index-pattern in Kibana in order to update
```

Comment:
In most cases probably directly after the import the refresh will not help as not much data is in the index yet. Only over time these fields will show up and as far as I understand every time a new dict with fields inside is added, the refresh is needed. Perhaps we should put this under the FAQ in something like "Fields show up a nested JSON in Kibana"?